client/src/components/admin/AdminDashboard.tsx

```
@@ -0,0 +1,123 @@
+              activeSection === 'verification' ? 'bg-purple-100 text-purple-700' : 'hover:bg-purple-50'
+            }`}
+          >
+            <ShieldCheck size={20} />
+            Verification Requests
+          </button>
+          <button
+            onClick={() => handleSectionChange('reports')}
+            className={`flex items-center gap-3 p-3 rounded-md text-gray-800 ${
+              activeSection === 'reports' ? 'bg-purple-100 text-purple-700' : 'hover:bg-purple-50'
+            }`}
+          >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)